ida-customizations/ida-course/exercise-week2.md

```diff
@@ -1,12 +1,26 @@
 ---
-title: "Week 2 Exercise"
+title: "Coding Exercise 1"
 order:
-description: From simple creation to practice
+description: Introducing the Final Exam
 tags:
   - deep-dive
 ---
 
-# Week 2 Exercise
+# Coding Exercise 1
+
+**Coding Exercise 1**, to adjust one line of code, is now accessible in your personal repo. Click [here](link) to go to the Academy repository.
+
+<HighlightBox type="note">
+
+The four Coding Exercises are worth 80% of your Final Exam score collectively. 
+
+You are free to attempt this exercise now, or you can wait until the Final Exam period officially begins on January 4th, 2024.
+
+</HighlightBox>
+
+When you complete the exercise you will be able to see your score directly, and will receive emailed feedback. You can repeat this exercise as often as you like if you want to improve your score.
```

Comment:
I am not sure there will be emailed feedback as often as there is a change of scores for instance.

Reply:
I've removed the emailed feedback mention, here and in the others. It can be a nice surprise for them ;-)